metrics/metrics.go

```diff
@@ -0,0 +1,32 @@
+package metrics
+
+import metrics "github.com/rcrowley/go-metrics"
+
+type Metrics struct {
+	registry metrics.Registry
+}
+
+const (
+	clientCounterName = "counter"
+)
+
+func New() (Metrics, error) {
```

Comment:
__[golint]__
* [golint] exported function New should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->